tools/system_libs.py

```diff
@@ -832,13 +833,13 @@ def can_build(self):
 
 class libc_extras(MuslInternalLibrary):
```

Comment:
maybe let's call it `extras_fastcomp` like the filename?

Reply:
This was always a fastcomp-only library.   I'm not changing that about it.   We could rename it but I'm not sure it worth it at this point with fastcomp going away.

Reply:
Makes sense, yeah, no point in polishing fastcomp stuff.